src/uwtools/tests/drivers/test_driver.py

```diff
@@ -397,6 +400,22 @@ def test_Driver_namelist_schema_default_disable(driverobj):
         assert driverobj.namelist_schema() == {"type": "object"}
 
 
+def test_Driver_output(config):
+    driverobj = ConcreteDriverTimeInvariant(config)
+    assert driverobj.output == {"foo": "/path/to/foo", "bar": ["/path/to/bar1", "/path/to/bar2"]}
```

Comment:
Since `ConcreteDriverTimeInvariant` implements a custom `output` method, we can test its return value.